tasks/test_jest.js

```diff
@@ -3,15 +3,18 @@ const { resolve } = require('path');
 module.exports = function (grunt) {
   grunt.registerTask('test:jest', function () {
     const done = this.async();
-    runJest().then(done, done);
+    runJest(resolve(__dirname, '../scripts/jest.js')).then(done, done);
   });
 
-  function runJest() {
+  grunt.registerTask('test:jest_integration', function () {
+    const done = this.async();
+    runJest(resolve(__dirname, '../scripts/jest_integration.js')).then(done, done);
```

Comment:
Can you update this to pass each of the segments of the path as individual arguments to `resolve` rather than hard coding the slashes? Otherwise this task will fail to run on windows, which doesn't use the same slashes.

e.g. `resolve(__dirname, '..', 'scripts', 'jest_integration.js')`

Would you mind also updating the scripts/jest path above?

Reply:
Sure 👍

Reply:
Btw, doesn't NodeJS on Windows support both slashes and backslashes?

Reply:
@azasypkin `require` works with forward slashes on any operating system, but file system commands via `fs` require paths with the native slash, which is one of the primary reasons why `resolve` exists in the first place as far as I know.

Do you have cygwin on your windows machine? It's entirely possible that resolves the slash issue at the OS level. It's also possible the latest versions of node have changed the way `resolve` works so slashes in path strings are converted as well.

In any case, if you've verified this works on windows, then there's nothing that needs to be changed here.

Reply:
> Do you have cygwin on your windows machine?

I just used standard `cmd`, NodeJS 6.12.2 and Windows 10 - all went well. Looks like NodeJS handles this internally.